api/organisations/subscriptions/licensing/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+from organisations.subscriptions.licensing.licensing import LicenceInformation
+
+
+class OrganisationLicence(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    organisation = models.OneToOneField(
+        "organisations.Organisation", related_name="licence", on_delete=models.CASCADE
+    )
+
+    content = models.TextField(blank=True)
+
+    def get_licence_information(self) -> LicenceInformation:
+        return LicenceInformation.parse_raw(self.content)
```

Comment:
I guess this implies that the licence information is still stored in the database in plain text. That's probably fine (and better than the alternative of having to decrypt every time we need access). 

I guess an (overkill perhaps) alternative would be adding some kind of time-based cache to hold the decrypted licence information so that we're only decrypting it once every 24h for example?

Reply:
We discussed this over standup and we've determined to leave it unencrypted for now.